scripts/ta_bin_to_c.py

```diff
@@ -37,6 +39,27 @@ def get_args():
     return parser.parse_args()
 
 
+def get_name(obj):
+    # Symbol or section .name might be a byte array or a string, we want a
+    # string
```

Comment:
very minor: suggest a single line comment:
`    # Symbol or section .name can be a byte array or a string, we want a string`